File: benchmark/results/java-multilang/run-004/a-mcp/output/billing/services/InvoiceRepository.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;

import java.util.List;
import java.util.Optional;

// InvoiceRepository service.

public class InvoiceRepository {

    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    public Optional<Invoice> findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-002/a-mcp/output/billing/services/invoice_repository.py
from typing import Optional
from billing.aggregates.invoice import Invoice

"""InvoiceRepository service."""
class InvoiceRepository:
    def create(self, input: Invoice) -> Invoice:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Invoice]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Invoice]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

